calico-cloud/networking/egress/egress-gateway-azure.mdx

```diff
@@ -0,0 +1,678 @@
+---
```

Comment:
Are we sure this is going in CC? I thought we'd been through this one already. @sridhartigera

Reply:
I think adding it for CC next is fine. I checked with @Glen-Tigera and he said it has been tested and is working.